### PR TITLE
Create metric-model-template.md

### PR DESCRIPTION
**Pull request by germonprez**
Signed-off-by: Matt Germonprez <germonprez@gmail.com>

### CODE REVIEW

**Comment by germonprez** [comment]
Please merge :) This is to start the new metrics model template.

File: resources/metric-model-template.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (GeorgLink) [inline resources/metric-model-template.md:1]:
```suggestion
# Metrics Model Template

This file is a placeholder for now.

```

**Review by GeorgLink — CHANGES_REQUESTED** [review]
Thanks @germonprez 

I suggest adding some text in the file so everyone knows what it is about when they look inside of it.

**Review by GeorgLink — APPROVED** [review]
LGTM - Thanks @germonprez

**Pull request merged**